buildmaster/H1_2JET_319GEV_290PB-1_DIF/metadata.yaml

```
@@ -0,0 +1,75 @@
+setname: 'H1_2JET_319GEV_290PB-1_DIF'
+
+nnpdf_metadata:
+  nnpdf31_process: "2JET"
+  experiment: "H1"
+
+arXiv:
+  url: "https://arxiv.org/abs/1611.03421"
+iNSPIRE:
+  url: "https://inspirehep.net/literature/1496981"
+hepdata:
+  url: "https://www.hepdata.net/record/ins1496981"
+  version: 1
+
+version: 1
+version_comment: "NA"
+
+implemented_observables:
+
+  - observable: {description: "double differential cross section w.r.t. q^2 and pT", label: '$\frac{d^2\sigma}{dq^{2}d\langle pT \rangle}$', units: "$pb/GeV^3$"}
+    observable_name: PTQ2
+    process_type: DIS_DIJET
+    ndata: 48
+    tables: [9, 10, 11, 12, 13, 14, 15, 16]
+    kinematics:
+      variables:
+        pT: {description: "mean transverse momentum", label: '$\langle pT \rangle$', units: "$GeV$"}
+        Q2: {description: "momentum transferred squared", label: '$q^2$', units: "$GeV^2$"}
+        sqrt_s: {description: "center of mass energy", label: '$\sqrt{s}$', units: "$GeV$"}
+      file: kinematics_dijet.yaml
```

```suggestion
      file: kinematics.yaml
```

there's no kinematics_dijet